server/server.cpp

```diff
@@ -513,7 +515,12 @@ int Server::exec(Cutelyst::Application *app)
     Q_D(Server);
     std::cout << "Cutelyst-WSGI starting" << std::endl;
 
-
+#ifdef Q_OS_LINUX
+    if (!d->userEventLoop) { //launchs Event loop only in case standalone app, not embedded
```

Comment:
This is missing qEnvironmentVariableIsSet("CUTELYST_QT_EVENT_LOOP") as cutelystd might be used with default event loop.
The comment is also a bid misleading, as I might embedded it to get better at LTO and still want EPool.